docs/http_protocol.md

```diff
@@ -86,7 +86,7 @@ Example of the overrides object:
 
 ## HTTP responses from RAG chat app endpoints
 
-The HTTP response is JSON for a non-streaming response, or [newline-delimited JSON](https://ndjson.org/home/) ("NDJSON"/"jsonlines") for a streaming response.
+The HTTP response is JSON for a non-streaming response, or [newline-delimited JSON](https://database.guide/what-is-ndjson/) ("NDJSON"/"jsonlines") for a streaming response.
```

Comment:
Eek, thanks for catching the link decay, guess ndjson let their domain expire. How about https://jsonlines.org/ ? That seems better maintained and is more of a dedicated resource than the database.guide site.